sqlalchemy_ibmi/base.py

```diff
@@ -205,51 +246,51 @@ class XML(sa_types.Text):
 
 
 class DB2TypeCompiler(compiler.GenericTypeCompiler):
+    """IBM i DB2 Type Compiler"""
```

Comment:
```suggestion
    """IBM i Db2 Type Compiler"""
```

The marketing people changed the brand from DB2 -> Db2 a couple years ago. It's pretty dumb, but might as well use the correct branding for any new code...